libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/permissions/DocumentPermissions.java

```diff
@@ -78,6 +79,8 @@ protected boolean getStandardPermissions(RequestedAction action) {
             case USERS:
             case CLEARING:
                 return PermissionUtils.isAdmin(user) || isModerator();
+            case WRITE_ECC:
+                return PermissionUtils.isAdmin(user);
```

Comment:
But aren't the `WRITE_ECC` things exactly the actions which are allowed to ECC_ADMIN and ADMIN? Thus this would be consistend and would work with future ECC information appended to other objects